### ISSUE TITLE
ortholinear

### ISSUE THREAD

**Issue by pvinis** [mw8/white_keyboard_layout#2]
awesome work. looks like you are trying to imitate an ortholinear keyboard by shifting the right hand one key. i was trying this layout today for a while. it feels better than qwerty of course.

have you tried using an ortholinear keyboard like ergodox? this middle part with the -+ button and @? etc, feels the same.

i wonder how your algorithm would change if you use the ergodox keyboard instead of a normal staggered keyboard.

**Comment by mw8** [comment]
It's funny you should bring this up, because I just ordered an ergodox in the latest offering on massdrop last week! This is a great idea. I will have to try this for ergodox when I get it.

**Comment by pvinis** [comment]
oh thats awesome. i am soon ordering mine from EZ. And after trying you layout for a bit, i think it would be awesome on it. on the laptop keyboard its a bit too crammed, but thats because of the laptop.

then i will wait for your report back :D

**Comment by pvinis** [comment]
did you get it? did you get it?!

**Comment by mw8** [comment]
It's delayed by massdrop until mid-September. :(

**Comment by mw8** [comment]
To late September. :(

**Comment by pvinis** [comment]
oh no.. that sucks.

btw use https://github.com/jackhumbert/qmk_firmware for programming the ergodox. its pretty awesome, if you dont know it. or start with a simple visual tool and use `qmk` later.

im looking forward. to you reporting back. i have been using my ergodox everyday this last week, finally. im happy :)

**Comment by pvinis** [comment]
did it arrive finally? :D

**Comment by mw8** [comment]
Yes, I got it and assembled it and I've been using it regularly. Through my usage I've been trying to get an idea of which keys should be reserved for escape, backspace, delete, tab, alt/option, ctrl, home, end, page up, page down, win/cmd, and arrow keys. All of the other keys (letters, numbers, punctuation, space) are represented correctly in the body of text that I use in the layout optimization, but those meta keys and movement keys are not represented adequately. So I'm still weighing how to proceed. What do you think?

I was looking at the [RSTHD ergodox layout](https://xsznix.wordpress.com/2016/05/16/introducing-the-rsthd-layout/) and I found it interesting that the E key was moved to the left thumb.

**Comment by pvinis** [comment]
wow havent seen that before. i will check it out!
did you use qmk to configure it? if yes, you should check other peoples configs like @algernon. he and others are using one-shot, tap-dance and other tricks to use fewer keys. so you can combine those modifiers etc.
https://asylum.madhouse-project.org/blog/2016/10/15/multi-purpose-keys/
check this out too. its from the same guy i mentioned above. he has some nice explanations and use cases about those tricks.

**Comment by xsznix** [comment]
@mw8 What layout do you use on your ErgoDox now, particularly w.r.t. modifiers and movement keys? IMO it's pretty difficult, if not impossible, to optimize those programmatically so it's all up to personal preference—for example, I like splitting the arrow cluster across hands but I know it would cause a lot of other people confusion.

I see that you've seen [my layout](https://github.com/xsznix/qmk_firmware/blob/master/keyboard/ergodox_ez/keymaps/keymap_rsthd.c#L35-L52), so I'm curious how your solution compares.

**Comment by Chocrates** [comment]
Hey @mw8, not sure if you are still watching this but did you end up continuing to use this layout on your ergodox?  
I am looking for a layout to use on my Kinesis and this looks interesting but not sure if it would be optimized.
